lib/jsonapi/authorization/authorizing_operations_processor.rb

```diff
@@ -17,6 +17,28 @@ class AuthorizingOperationsProcessor < ::ActiveRecordOperationsProcessor
       set_callback :remove_to_many_relationship_operation, :before, :authorize_remove_to_many_relationship
       set_callback :remove_to_one_relationship_operation, :before, :authorize_remove_to_one_relationship
 
+      [
+        :find_operation,
+        :show_operation,
+        :show_related_resource_operation,
+        :show_related_resources_operation,
+        :create_resource_operation,
+        :replace_fields_operation
+      ].each do |op_name|
+        set_callback op_name, :after, :authorize_include_directive
+      end
+
+      def authorize_include_directive
+        resources = if @result.respond_to?(:resources)
+                      @result.resources
+                    else
+                      [@result.resource]
```

Comment:
@valscion this had not be added in that branch and it's causing issue on our spec still. Any reason for it?

``` ruby
resources = Array.wrap(
  if @result.respond_to?(:resources)
    @result.resources
  elsif @result.respond_to?(:resource)
    @result.resource
  end
)
``` 
still fixes it.